test/TestInvestorLedger.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.4.11;
+
+import "truffle/Assert.sol";
+import "truffle/DeployedAddresses.sol";
+import "../contracts/loans/InvestorLedger.sol";
+import "../contracts/tokens/PrintableToken.sol";
+
+
+contract TestInvestorLedger {
+    uint256 totalLoanNeeded = 100;
+    uint16 interestPermil = 200;
+    uint256 EMPTY = 0;
+    uint256 PRINT_VALUE = 100;
+    PrintableToken collateralToken = createMockToken("collateralToken");
+    PrintableToken loanToken = createMockToken("loanToken");
+    TestPerson investor = new TestPerson(loanToken, collateralToken);
+    TestPerson borrower = new TestPerson(loanToken, collateralToken);
+    InvestorLedger.Ledger testLedger = InvestorLedger.openAccount(
+        collateralToken, loanToken, address(borrower), totalLoanNeeded, interestPermil);
+
+    function createMockToken(string memory tokenName)  returns (PrintableToken) {
+        uint256 decimalPoints = 0;
```

Comment:
Why the variable?

Reply:
if it was "new PrintableToken("tokenName", 0, "adsf", 1234)" I'd have to guess, what 0 and 1234 are. Maybe overkill

Reply:
This is test code - it's tightly coupled with the code under test, so we can expect whoever reads this to at least have the definition of PrintableToken::PrintableToken in a tab close by.